doc/DOCUMENTATION.org

```diff
@@ -421,7 +421,7 @@ contribution guidelines in [[./CONTRIBUTE.org][CONTRIBUTE.org]].
 
 ** Example: Themes Megapack example
 This is a simple =contrib= configuration layer listing a bunch of themes, you
```

Comment:
I guess the whole paragraph is out of date, I'm going to merge this fix for now.